### ISSUE TITLE
about logits softmax

### ISSUE THREAD

**Issue by momaaron** [Emerge-Lab/PufferDrive#410]
I found that in pure evaluation and reasoning, there is no corresponding configuration to set greedy selection, that is, argmax. Shouldn't this support a parameter configuration?

**Comment by KZL-7** [comment]
<img width="612" height="101" alt="Image" src="https://github.com/user-attachments/assets/a38c1d06-692b-429b-a032-5ec085c80f74" />And the C code is written to select argmax, but the function used is actually softmax. Why is that?

**Comment by eugenevinitsky** [comment]
no particular reason, feel free to open a bug fix, thanks!
